packages/braid-design-system/src/lib/components/useToast/useToast.docs.tsx

```diff
@@ -428,9 +428,9 @@ const docs: ComponentDocs = {
       description: (
         <>
           <Text>
-            {`A toast message can be dismissed via the close button or automatically after ${
-              toastDuration / 1000
-            } seconds.`}
+            A toast message can be dismissed via the close button or
+            automatically after a timeout period depending on its content
+            length.
```

Comment:
I think we should level these docs up to capture the algorithm being used. It'd be good to interpolate the time values as well. We cna pair on the phrasing if you like.

Reply:
Workshopped the language to detail the algorithm used, but happy to pair on this. I'm now thinking it's probably easier to describe the minimum as 6 seconds, as that's the minimum time in practice. Updated the variables to reflect this, thought the calculation will give the same result